.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+### Summary
```

Comment:
@jeremywiebe a good point. We probably don't even need github issues / feature requests for this repo. I'll add a couple Readmes to both directories describing the files and their purpose.